README.md

```diff
@@ -18,8 +18,8 @@ Repository to describe, develop, document and test the ClickToDial API family
 
 ## Meetings
 * Meetings are held virtually
-* Schedule: tbd
-* Meeting link: tbd
+* Schedule: bi-weekly (odd weeks), Wednesday, 1 PM UTC+8
```

Comment:
If based on UTC, then maybe 05:00 UTC is more clear (local time zone could be mentioned in addition). 
BTW: will it change with daylight saving time, or is fixed to UTC? So will it be 07:00 CEST and later 06:00 CET?